pyproject.toml

```diff
@@ -22,6 +22,10 @@ classifiers = [
 python = ">=3.8, <3.11"
 aiohttp = "^3.8.1"
 msgpack = "^1.0.3"
+grpcio = "^1.48.1"
+grpcio-tools = "^1.48.1"
+protobuf = "^3.20.1"
+grpcio-testing = "^1.48.1"
```

Comment:
Add grpcio testing in dev dependencies

Reply:
only `grpcio-testing = "^1.48.1"` ??
```
from grpc import StatusCode
from grpc_testing import server_from_dictionary, strict_real_time
```
I've imported `grpc` in the tests as well

Reply:
as discussed offline...
yes only `grpcio-testing = "^1.48.1"`

Reply:
need help... not able to properly add it

Reply:
> > @chromevoid update the version as major/minor upgrade. @vigith this should be major right? Or should minor be okay?
> 
> where to update the version? which file?
> 
> toml??
> 
> ```
> [tool.poetry]
> name = "pynumaflow"
> version = "0.1.1"
> ```

run `poetry minor update` ?

Reply:
discussed offline